cnf-certification-test/networking/suite.go

```diff
@@ -124,6 +128,22 @@ func testListenAndDeclared(env *provider.TestEnvironment) {
 	}
 }
 
+func testNodePort(env *provider.TestEnvironment) {
+	badNamespaces := []string{}
+	for _, put := range env.Pods {
```

Comment:
You should test namespaces but you're iterating thorugh pods... why? This is the for loop of the implementation of this TC in the old repo:
```
		for _, ns := range env.NameSpacesUnderTest {
			ginkgo.By(fmt.Sprintf("Testing services in namespace %s", ns))
```
For this repo you should use `env.Namespaces`.

Reply:
My mistake!
I thought that I need to use in serviceAccount , so this is the reason that I run on pods and no namespace.